api/spec/controllers/spree/api/v1/checkouts_controller_spec.rb

```diff
@@ -296,6 +320,12 @@ module Spree
         api_put :next, id: order.to_param, order_token: order.guest_token, order: {}
         expect(json_response["errors"]["base"]).to include(Spree.t(:no_payment_found))
       end
+
+      def send_request(params = {})
```

Comment:
Unused method argument - `params`. If it's necessary, use `_` or `_params` as an argument name to indicate that it won't be used. You can also write as `send_request(*)` if you want the method to accept any arguments but don't care about them.